docs/metabase.md

```diff
@@ -0,0 +1,24 @@
+# Setting up Metabase on AWS
+
+The instructions on Metabase docs are already quite good. This document is just a few salient points to assist the docs to fit with our infrastructure.
+
+Refer largely to the Metabase docs and just use this for clarification on a few settings and tweaks.
+
+* In the Quick Launch section, choose a descriptive name, like:
+
+* In the Upload Code section, don't choose the default S3 URL. Download Metabase from here and upload it manually.
+
+* In the Network section after launching the app, select the "Default VPC".
+
+
+Check:
+
+Visibility: Public
+Load Balancer Subnets: Select all
+Pulibc IP Address: Yes
+Instance Subnets: Select all
+Database Subnets: Select all
```

Comment:
I would love to bullet-list these for readability. I suspect we haven't because the top-level steps are bullets? How about something like this?

### 1. Quick Launch

In the Quick Launch section, choose...

### 2. Upload Code

Moar instructions

### 3. Network

Instructions

* with
* a
* list

---
I love formatting documentation, can you tell? 😅